extension.json

```diff
@@ -5,6 +5,5 @@
     "Carlo Caione <carlo.caione@gmail.com>"
   ],
   "description": "NeoSolarized Zed theme",
-  "repository": "https://github.com/carlocaione/NeoSolarized.zed",
-  "themes": ["themes/neosolarized"]
```

Comment:
We haven't updated the docs yet, but we no longer need the list of files specified in the `extension.json`.